### PR TITLE
Zc/cc landing page

### PR DESCRIPTION
**Pull request by corps**
Final integration piece of community project first pass.

### CODE REVIEW

File: etna/lib/etna/route.rb
Comment (corps) [inline etna/lib/etna/route.rb:129]: api calls should still just 403 here, since there is not a browser context to request guest access with.

File: janus/lib/client/jsx/cc-view.jsx
Comment (corps) [inline janus/lib/client/jsx/cc-view.jsx:49]: These two lines are changed once we have the janus admin ui piece in place, this just allows simpler testing with inline copy of the ipi cc.  Will remove before merging.

File: etna/lib/etna/route.rb
Comment (coleshaw) [reply etna/lib/etna/route.rb:129]: Ah, that makes sense, since it'll be from an XHR request.

Does this mean that the client libraries in `etna-js` will have to do the right messaging or redirects to the code of conduct page? For example, if someone opens Timur and from the root view clicks into IPI, I think the first time they'll hit an endpoint that triggers this code is when the client app does a fetch from Magma `/retrieve` ... ? So would that have to catch the 403 and then redirect the app? (same problem whether it's the 302 or the 403)

File: janus/lib/server/controllers/admin_controller.rb
Comment (coleshaw) [inline janus/lib/server/controllers/admin_controller.rb:170]: With #874 , the `contact_email` can be `""`, which I think would evaluate to true in this case, but then try to send an e-mail to an empty string address? Should this do a check for `&.empty?` here, to handle that case?

**Review by coleshaw — APPROVED** [review]
Looks good! Just the one question about handling the 403, which doesn't affect this PR, but would affect additional integration work that's needed.

File: etna/lib/etna/route.rb
Comment (corps) [reply etna/lib/etna/route.rb:129]: Not quite -- check timur/server.rb:

```
    with auth: { user: { can_view?: :project_name } } do

...
      # remaining view routes are parsed by the client and must also be set there
      get ':project_name', as: :project do
        erb_view(:client)
      end

      get ':project_name/*view_path', as: :project_view do
        erb_view(:client)
      en
```

So in other words, even the document views will trigger this from the browser.

File: janus/lib/server/controllers/admin_controller.rb
Comment (corps) [reply janus/lib/server/controllers/admin_controller.rb:170]: right right -- I'll check for empty here.

File: etna/lib/etna/route.rb
Comment (coleshaw) [reply etna/lib/etna/route.rb:129]: Ah, nice, I missed the top of the block when I was scanning :facepalm: 

:+1:

Comment (graft) [reply etna/lib/etna/route.rb:129]: Hmm, Janus server.rb currently has: `get '/*views' do erb_view(:client) end`; the project-specific page is unguarded and wouldn't be caught this way, they would just show up and get a broken/empty page.

Comment (corps) [reply etna/lib/etna/route.rb:129]: Right, so I need to swap out the 'get project' endpoint for 'get projects' on the cc view, that way they can still pull open projects without permission and see the page correctly, making the change.

**Comment by corps** [comment]
Note to self: do not merge until after janus editing lands, so I can remove the fake cc text and test e2e.

**Pull request merged**
